ppr-ui/src/components/common/Breadcrumb.vue

```diff
@@ -86,6 +86,19 @@ export default defineComponent({
         const roleBasedBreadcrumbTitle = breadcrumbsTitles[
           getRoleProductCode(getUserRoles.value, getUserProductSubscriptionsCodes.value)
         ]
+        const allTombstoneBreadcrumbs = [
+          tombstoneBreadcrumbDashboard,
+          tombstoneBreadcrumbDischarge,
+          tombstoneBreadcrumbRenewal,
+          tombstoneBreadcrumbAmendment,
+          tombstoneBreadcrumbRegistration,
+          tombstoneBreadcrumbSearch,
+          tombstoneBreadcrumbSearchConfirm
+        ]
+        if (localState.isStaff) {
+          const staffBreadcrumbText = 'Staff Dashboard'
+          allTombstoneBreadcrumbs.forEach((val) => { val[0].text = staffBreadcrumbText })
```

Comment:
Might not need a const if just used locally here,
I was thinking if it was defined in the BreadCrumb resource or something 😁 
All good either way though, no change required 👍

Reply:
oh sorry didn't see your comments before making some other changes, but I think the updated code has pretty much the same idea, I'm just using `for..of` instead of `foreach`

Reply:
You bet! All good sir 👍

Reply:
removed the redundant `const`!